transports/sqs/delayed-delivery.md

```diff
@@ -0,0 +1,145 @@
+
+```mermaid
+sequenceDiagram
+    participant D as Destination
+    participant F as Destination-delay.fifo
+    F ->>+ F: Timeout due
+    F ->> D: Send with remaing delay
+    Note left of D: Original message
+    F ->>- F: Delete Delayed Message failed
+    Note right of F: Network outage
+    F ->>+ F: Timeout due
+    F ->> D: Send with remaing delay
+    F ->>- F: Delete Delayed Message
+    Note left of D: Duplicate message
+```
+
+### Example
+
+Below an example of a delayed delivery less or equal to 900 seconds.
+
+```mermaid
+graph LR
+
+subgraph Example: delay of 14 min and 5 seconds
+
+sender
+destination
+
+sender .-> |T1: Delay with 845sec| destination
+destination --> |"T2: fa:fa-hourglass-half 845sec"| destination
+
+end
+```
+
+14 min and 4 seconds are in total 845 seconds. This is less than 900 seconds and thus will be directly delayed to the destination with a `DelaySeconds` value of 845 seconds. No message attribute header will be used.
+
+Below an example of a delayed delivery greater than 900 seconds.
+
+```mermaid
+graph LR
+
+subgraph Example: delay of 32 min and 5 seconds
+
+sender
+fifo(destination-delay.fifo)
+destination
+
+sender .-> |T1: Delay with 1925sec| fifo
+fifo --> |"T2: fa:fa-hourglass-half Delay with 900sec"| fifo
+fifo --> |"T3: fa:fa-hourglass-half Delay 900sec"| fifo
+fifo .-> |"T3: Send to destination"| destination
+destination --> |"T4: fa:fa-hourglass-half Delay with 125sec"| destination
+
+end
+```
+
+32 min and 5 seconds are in total 1925 seconds. This will lead to two 900 seconds cycles on the FIFO queue and one delayed delivery on the destination queue with the remaining timeout of 125 seconds (handover between FIFO queue and input queue).
+
+## Cost considerations
+
+Each non-sendonly endpoint with the unrestricted delayed delivery enabled operates one delayed message consumer that applies [long-polling](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-long-polling.html) with an interval of 20 seconds. [Satellites](/nservicebus/satellites/) will increase the number of delayed message consumers. The following example shows a cost calculation for a message timeout for a year, not taking into account the long-polling costs.
+
+Assuming the following SQS prices:
+
+Price per 1 Million Requests after Free Tier (Monthly)
+
+| Standard Queue | $0.40 ($0.00000040 per request) | 
+| FIFO Queue     | $0.50 ($0.00000050 per request) |
+
+More up-to-date information on pricing can be found on the [SQS pricing page](https://aws.amazon.com/sqs/pricing/).
+
+Delaying a message for a year, 365 days or 31536000 (365*24*60*60) seconds, and two operations (dequeue and requeue) per delay interval of 900 seconds. The delayed message will need to go through 35040 delay cycles (3153600 sec / 900 sec) which leads to 70080 queue operations. The end costs would be
```

Comment:
31536000 -> 31,536,000 reads better, and so on for the other numbers in the same sentence